inc/endpoints/class-post-select-controller.php

```diff
@@ -140,7 +140,12 @@ public function get_items( $request ) {
 
 		$query_args = apply_filters( 'hm_gb_tools_post_select_query_args', $query_args );
 
+		do_action( 'hm_gb_tools_post_select_before_query' );
```

Comment:
I wonder if it would be worth passing $query_args as a param to these actions, so that you can run logic conditionally based on the query?

Reply:
Yea having the option to do that conditional logic would help. Let me add that.